plugins/alarmclock/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/alarmclock/AlarmClockSpeech.java

```diff
@@ -270,8 +271,12 @@ public String getRemainingTimerDelay(String[] params) {
 			return "Timer " + params[3] + " rings in " + remDelay[0] + " hours and " + remDelay[1] + " minutes and "
 					+ remDelay[2] + " seconds";
 		} catch (NoSuchElementException e) {
-			this.logger.log(Level.SEVERE, "Exception was thrown:", e);
+			this.logException(e);
 			return ELEMENTNOTFOUND;
 		}
 	}
+	
+	private void logException(Exception e) {
+		this.logger.error("An " + e.toString() + " was thrown!");
```

Comment:
use https://www.slf4j.org/api/org/slf4j/Logger.html#error(java.lang.String,%20java.lang.Throwable) to log exceptions, this way the complete stack trace is logged